otherlibs/threads/thread.mli

```diff
@@ -40,7 +40,8 @@ val self : unit -> t
 val id : t -> int
 (** Return the identifier of the given thread. A thread identifier
    is an integer that identifies uniquely the thread.
-   It can be used to build data structures indexed by threads. *)
+   It can be used to build data structures indexed by threads.
+   @since 4.03 *)
```

Comment:
indeed it used to be `external id` before, but I don't think that distinction matters for user code.